setup.py

```diff
@@ -27,4 +27,9 @@
     packages=find_packages(exclude=["contrib", "docs", "tests"]),
     install_requires=install_reqs,
     package_dir={"shimmingtoolbox": "shimmingtoolbox"},
+    entry_points={
+        'console_scripts': [
+            'st_script_nifti = examples.script_nifti:main',
```

Comment:
i would not add example scripts as entry points-- these are not the kind of scripts that users will call during regular usage

Reply:
There's a good reason for this. It means you can do `pip install .` on every platform and have `st_script_nifti` become a runnable command.  Windows is the odd duck out here: it *needs* `console_scripts` in order for this to work. Without that, on Windows you have to run the script with `python examples/script_nifti.py` or `python -m shimmingtoolbox.examples.script_nifti` and that unfortunately has subtle implications about the import path.

I don't think these scripts should stay here in `console_scripts` forever but it's a useful starting point that gets everyone on the same page.

Reply:
But I guess it's not clear if *this* particular script is part of the CLI or not. Is it going to be a tool or an example, @gaspardcereza ?